sale_exception_print/wizard/sale_exception_confirm.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in module root
+# directory
+##############################################################################
+from openerp import models, api
+
+
+class SaleExceptionConfirm(models.TransientModel):
+
+    _inherit = 'sale.exception.confirm'
+
+    @api.one
+    def action_confirm(self):
```

Comment:
Nico, esto no esta muy bueno porque se está sobre escribiendo toda la función action confirm, sin llamar a super, entonces puede ser que esto anule la funcionalidad que agreguen otros módulos según el orden en que odoo los cargue. Por ejemplo puede ser que se deje de ejecutar el "action_confirm" de ingore_approve

Me parece mas bien que haría algo así:
def action_confirm(self):
    if self.ignore and self._context.get('print_exceptions', False):
        self.sale_id.ignore_exception_print = True
        # con esto limpiamos el ignore para que no super no ignore todo
        self.ignore = False
    return super(SaleExceptionConfirm, self).action_confirm()